sunpy/sun/sun.py

```diff
@@ -51,124 +59,28 @@ def solar_semidiameter_angular_size(t='now'):
     """
     # Import here to avoid a circular import
     from sunpy.coordinates import get_sunearth_distance
-    solar_semidiameter_rad = (constants.radius.to(u.AU)) / get_sunearth_distance(t)
+    solar_semidiameter_rad = constants.radius / get_sunearth_distance(t)
     return Angle(solar_semidiameter_rad.to(u.arcsec, equivalencies=u.dimensionless_angles()))
 
 
 @add_common_docstring(**_variables_for_parse_time_docstring())
 def position(t='now'):
     """
-    Returns the position of the Sun (right ascension and declination) on the
-    celestial sphere using the equatorial coordinate system.
-
-    .. warning::
-        This function does not use the coordinates framework, so there may be inconsistencies.
+    Returns the apparent position of the Sun (right ascension and declination) on the
+    celestial sphere using the equatorial coordinate system, referred to the true equinox of date.
+    Corrections for nutation and aberration (for Earth motion) are included.
 
     Parameters
     ----------
     t : {parse_time_types}
         A time (usually the start time) specified as a parse_time-compatible
         time string, number, or a datetime object.
-
-    Notes
-    -----
-    Based on formulae in "Astronomical Formulae for Calculators" by Meeus
     """
-    ra = true_rightascension(t)
-    dec = true_declination(t)
+    ra = apparent_rightascension(t)
+    dec = apparent_declination(t)
     return ra, dec
```

Comment:
Should this return a `SkyCoord`?

Reply:
In principle, yes, but the appropriate frame doesn't currently exist in Astropy.  The frame needs to be `PrecessedGeocentric` plus nutation corrections (i.e., true equinox of date instead of mean equinox of date).  So, for now, a tuple is what we can do.